src/DataStructures/SpinWeighted.hpp

```diff
@@ -212,7 +215,12 @@ struct SpinWeighted<T, Spin, true> {
 
 template <typename T, int Spin>
 void SpinWeighted<T, Spin, true>::pup(PUP::er& p) noexcept {
-  data_.pup(p);
+  p | data_;
+}
+
+template <typename T, int Spin>
+void SpinWeighted<T, Spin, false>::pup(PUP::er& p) noexcept {
```

Comment:
can you combine the two `pup` functions into one by adding the bool as template parameter?

Reply:
I tried that first, but because the pup is individually declared in the template specializations, the definition must be similarly specialized. (doesn't build otherwise)